### ISSUE TITLE
Do not understand program.

### ISSUE THREAD

**Issue by 23bca312** [vtcbca/csv-file-program-23bca312#2]
I understand how to create and insert record from sqlite3 but how to export table in csv file? 

Please explain and help me to solve this problem.

**Comment by 23bca307** [comment]
to export table into csv file use dump command

**Comment by 23bca312** [comment]
How to apply dump command.

**Comment by 23bca307** [comment]
.dump <tablename> > <filename>.csv 

This is the syntax but replace the content written in < > . i am not sure but try this solution.

**Comment by 23bca261** [comment]
to export table data into csv file , you have to use .output command.

**Comment by 23bca312** [comment]
Please explain with example.

**Comment by 23bca261** [comment]
ok.. let me explain you...!!!

for example your table name is stud. and you want to export its data into "stud.csv" file. you have to enter below command.

.mode column
.mode csv
.output stud > stud.csv

**Comment by 23bca312** [comment]
Is this command is given after creating insert command.

**Comment by 23bca261** [comment]
yes... you have to apply this command after creating and inserting record.

one main point is after giving above command, write select query.

select * from stud.
